app/Kconfig

```diff
@@ -7,4 +7,21 @@ config APP_DATA_LOGGING_SYNC_INTERVAL
 		This configures the interval at which the logging system flushes everything to disk
 		and hence determines the maximum duration of data lost on power cuts.
 
+choice
```

Comment:
Please add a choice name. I suggest `APP_PRIMARY_IMU`, but anything similar is fine.